pkg/apis/azuredisk/v1beta2/types.go

```diff
@@ -521,3 +521,36 @@ type ListSnapshotsResult struct {
 	// +optional
 	NextToken string `json:"next_token,omitempty"`
 }
+
+type AzDiskDriverConfiguration struct {
+	Controller ControllerConfiguartion `json:"controller"`
+	Node NodeConfiguration `json:"node"`
+	// The namespace where driver related custom resources are created
+	DriverObjectNamespace string `json:"diverObjectNamespace"`
+	// Boolean field to indicate whether this instance is being used for sanity or integration tests
+	IsTestRun bool `json:"isTestRun"` // not sure if it belongs to controller
```

Comment:
This is a test parameter used by the sanity and integration tests. I'd rather not include this in the configuration (making it public). Let's leave it as a command-line parameter only for now.